boards/pjrc/teensy4/board.yml

```diff
@@ -0,0 +1,9 @@
+boards:
+  - name: teensy40
+    vendor: pjrc
+    socs:
+      - name: mimxrt1062
+  - name: teensy41
+    vendor: pjrc
+    socs:
+      - name: mimxrt1062
```

Comment:
seems like board revisions would be appropriate here?

Reply:
I have no idea, this is not even my/our(NXP) board, revisions existed in v1 and whoever added this board  did not use them, so I just ported what they have, not doing anything else with this board

Reply:
Myself and others have been doing conversions and using board revisions regardless of vendor, that's kind of the point in hwmv2

Reply:
What I'm saying is I don't know if this is a revision, the person who added the board opted not to use a revision, even though it was possible, so I don't want to do something they didn't intend

Reply:
I just looked on their website and they have this comparison table 

![image](https://github.com/zephyrproject-rtos/zephyr/assets/110566814/ef64a396-b070-4440-a697-5b70e8756ae8)

I am thinking its not even revisions, just two different boards, I am not comfortable reworking this definition to use revisions, I dont even have the hardware

Reply:
@bdkrae is the 4.0 and 4.1 a revision of this board?

Reply:
@decsny is wright, 4.0 and 4.1 are two different boards, not revisions.